lib/lotus/generators/application/container/config/.env.development.tt

```diff
@@ -1,2 +1,14 @@
 # Define ENV variables for development environment
+<%- if config[:adapter] == 'mysql' %>
+<%= config[:app_name].to_env_s %>_DATABASE_URL="mysql://localhost/<%= config[:app_name] %>_development"
```

Comment:
@Linuus Can you please do the same here? I agree with @joneslee85 : it's better to resolve it in `container.rb`. Also because we'll reuse this code in the future for other generators.